mu/wheels/__init__.py

```diff
@@ -19,6 +19,7 @@
     ("pyserial", "pyserial==3.4"),
     ("qtconsole", "qtconsole==4.7.4"),
     ("nudatus", "nudatus>=0.0.3"),
+    ("esptool", "esptool"),

```

Comment:
Looks like esptool follows semver? v3.0 was released in November, I assume that version has been tested and works fine?

While I don't think we need to look a specific version we can probably set it to only install version 3.x, in case future (or previous) versions introduce breaking changes.